core/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@umaprotocol/core",
+  "version": "1.0.0",
+  "description": "UMA smart contracts and unit tests",
+  "private": true,
+  "dependencies": {
+    "@openzeppelin/contracts": "3.0.0",
+    "minimist": "^1.2.0",
+    "inquirer": "^7.0.4",
```

Comment:
inquirer-through-moment are all CLI deps that I want to refactor out in #1728

Reply:
Why not just leave them out, then? They should still work because the deps are listed at the root, right?

Reply:
Either way is fine -- just seemed easier to not have to add these and then remove them later.

Reply:
> Why not just leave them out, then? They should still work because the deps are listed at the root, right?

Good point, I'll remove them since I will remove them later